packages/siopv2-oid4vp-op-auth/src/types/machine/index.ts

```diff
@@ -101,9 +103,10 @@ export enum Siopv2MachineGuards {
   createContactGuard = 'Siopv2CreateContactGuard',
   hasContactGuard = 'Siopv2HasContactGuard',
   hasAuthorizationRequestGuard = 'Siopv2HasAuthorizationRequestGuard',
+  canDoGetSelectableCredentialsGuard = 'Siopv2CanDoGetSelectableCredentialsGuard',
   hasSelectedRequiredCredentialsGuard = 'Siopv2HasSelectedRequiredCredentialsGuard',
   siopOnlyGuard = 'Siopv2IsSiopOnlyGuard',
-  siopWithOID4VPGuard = 'Siopv2IsSiopWithOID4VPGuard',
+  canDoSelectCredentialsGuard = 'Siopv2CanDoSelectCredentialsGuard',
```

Comment:
Why would you rename something that was more clear about the protocols in play to this?

Reply:
Reverted, see later comment